src/components/PostComment.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState, useContext} from "react";
+import { postComment } from "../api";
+import { UserContext } from "../contexts/LoggedInUserContext";
+
+export default function PostComment({ review_id, addComment }) {

```

Comment:
You can submit the same comment twice if you press submit a second time, is this something we can avoid?

I'd also check that you can't press the submit button a second time before the first finishes posting, looks like you have something for this in your catch block of postComment you could use.